src/scheduler/parse.cpp

```diff
@@ -1093,43 +861,6 @@ preempt_cmp(const void *p1, const void *p2)
 	}
 }
 
-/**
- * @brief
- *	       Frees the memory for the corresponding sort_info structure variable given.
- *
- * @param[in]	si_type	- type of sort_info structure
- *
- * @return void
- *
- */
-void
-free_sort_info(enum sort_info_type si_type)
-{
-	struct sort_info *so_info;
-
-	switch (si_type) {
-		case PRIME_SORT:
-			so_info = conf.prime_sort;
-			break;
-		case NON_PRIME_SORT:
-			so_info = conf.non_prime_sort;
-			break;
-		case PRIME_NODE_SORT:
-			so_info = conf.prime_node_sort;
-			break;
-		case NON_PRIME_NODE_SORT:
-			so_info = conf.non_prime_node_sort;
-			break;
-		default:
-			return;
-	}
-
-	if (so_info != NULL) {
-		int i;
-		for (i = 0; i < MAX_SORTS; i++) {
-			if (so_info[i].res_name != NULL)
-				free(so_info[i].res_name);
-		}
-		free(so_info);
-	}
-}
+peer_queue::peer_queue(const char *lqueue, const char *rqueue, const char *rserver): local_queue(lqueue), remote_queue(rqueue), remote_server(rserver) {}
+timegap::timegap(time_t from, time_t to): from(from), to(to) {}
+dyn_res::dyn_res(const char *res, const char *cmdline, const char *fname): res(res), command_line(cmdline), script_name(fname) {}
```

Comment:
if you're not going to define them in the header file, i would put `{` and `}` on their own lines.